pyproject.toml

```diff
@@ -32,7 +32,7 @@ maintainers = ["GoDaddy <oss@godaddy.com>"]
 name = "tartufo"
 readme = "README.md"
 repository = "https://github.com/godaddy/tartufo/"
-version = "3.2.1"
+version = "3.2.2"
```

Comment:
I believe you need not update the version of tartufo as part of bug fix or implementation PR. This will be updated when a release is created

Reply:
Good catch, Esha! Anthony, can you revert the two version changes (here and below) she called out?

Reply:
Thank you for the review. I've reverted the change.